programs/ssl/ssl_server2.c

```diff
@@ -2652,7 +2654,7 @@ int main(int argc, char *argv[])
         }
         key_cert_init = 2;
 #endif /* MBEDTLS_RSA_C */
-#if defined(MBEDTLS_ECDSA_C)
+#if defined(MBEDTLS_PK_CAN_ECDSA_SIGN) || defined(MBEDTLS_CAN_ECDSA_VERIFY)
```

Comment:
Minor, cosmetic: I think `SIGN || VERIFY` can replaced with `SOME`. Or did you mean `&&`?

Reply:
You are right. I think that I wrote it this way because at the time in which I developed this we still had the `SOME` "variant" only available in `pk_wrap.h` and not in `pk.h` as it is now. I will fix this ;)